### PR TITLE
Document Tasmota cover and fan

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Document Tasmota cover and fan

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/tasmota.markdown [hunk 1]
```diff
@@ -24,14 +26,16 @@ This integration allows you to control [Tasmota](https://tasmota.github.io/docs/
...
-- Relays will be added as Home Assistant `switch` entities, if `SetOption30 = 0`. If `SetOption30 = 1`, relays will be added as `light` entities.
-- Sensors will be added as Home Assistant `sensor` entities.
-- Switches will be added as Home Assistant `binary_sensor` entities or `automation triggers` depending by the `switchmode` used when `SetOption114` is enabled.
-- Buttons will be added as Home Assistant `automation triggers` when `SetOption73` is enabled.
+- Tasmota Buttons will be added as Home Assistant `automation triggers` when `SetOption73` is enabled.
+- Tasmota Lights will be added as Home Assistant `light` entities. Single channel Dimmers, RGB lights, RGB lights with Color Temperature control and RGB lights with White control are supported.
+- Tasmota Relays will be added as Home Assistant `switch` entities, if `SetOption30 = 0`. If `SetOption30 = 1`, relays will be added as `light` entities.
+- Tasmota Sensors will be added as Home Assistant `sensor` entities.
+- Tasmota Shutters will be added as Home Assistant `cover` entities.
```
Comment (effelle) [inline source/_integrations/tasmota.markdown]: Please note `Shutter mode 5` (configuration with a single relay and a PWM) is not yet supported by HaTasmota. My bad, I need to finish the shutter mode for Tuya MCU to see what is the best way to implement it on the integration topic.

File: source/_integrations/tasmota.markdown [hunk 2]
```diff
@@ -24,14 +26,16 @@ This integration allows you to control [Tasmota](https://tasmota.github.io/docs/
...
+- Tasmota Switches will be added as Home Assistant `binary_sensor` entities or `automation triggers` depending by the `switchmode` used when `SetOption114` is enabled.
 - The integration will also create up to eight Status Sensors, each one with a different information. Please note all the Status Sensors are disabled by default.
+- The fan funciotnality in Tasmota devices with module configured as iFan02 or iFan03 will be added as Home Assistant `fan` entities.
```
Comment (effelle) [inline source/_integrations/tasmota.markdown]: Typo (funciotnality).

**Review by effelle — CHANGES_REQUESTED** [review]

File: source/_integrations/tasmota.markdown (hunk 1 above)
Comment (emontnemery) [reply source/_integrations/tasmota.markdown]: OK, how to express this limitation? Should we say shutters which are not `mode 5` and not Tuya are supported?

Comment (effelle) [reply source/_integrations/tasmota.markdown]: Shutter mode 5 is rarely used but for TuyaMCU will be perfect. I have no idea when I will be able to add it on Tasmota, even so will be not on 9.2. Perhaps if there will be a 9.2.1?
How about adding:
```
Currently only Shutter modes 1 to 4 are supported.
```

Comment (emontnemery) [reply source/_integrations/tasmota.markdown]: Thanks, updated.

**Review by effelle — APPROVED** [review]

**Review by frenck — APPROVED** [review]
Thanks, @emontnemery 👍 

Happy Holidays! 🎄

**Pull request merged**